spec/stop_visits.schema.json

```diff
@@ -29,12 +28,20 @@
       }
     },
     {
-      "name": "stop_sequence",
+      "name": "trip_stop_sequence",
       "type": "integer",
-      "title": "positive integer referencing GTFS stop_times.stop_sequence",
-      "description": "Ordered stop the vehicle is serving on a particular trip. References GTFS",
+      "description": "The actual order of stops visited within a performed trip. The values must start at 1 and must be consecutive along the trip.<br/> Example: A bus departs the first stop and detours around the second and third scheduled stops, visiting one unscheduled stop and resuming regular service at the 4th scheduled stop. The `scheduled_stop_sequence` is [1, null, 4], and the `trip_stop_sequence` is [1, 2, 3]."
```

Comment:
Missing a comma at the end of this line to separate `description` from `constraints`